replica/distributed_loader.cc

```diff
@@ -267,14 +267,18 @@ distributed_loader::reshape(sharded<sstables::sstable_directory>& dir, sharded<r
 // Loads SSTables into the main directory (or staging) and returns how many were loaded
 future<size_t>
 distributed_loader::make_sstables_available(sstables::sstable_directory& dir, sharded<replica::database>& db,
-        sharded<db::view::view_update_generator>& view_update_generator, fs::path datadir, sstring ks, sstring cf) {
+        sharded<db::view::view_update_generator>& view_update_generator, bool to_staging, sstring ks, sstring cf) {
 
```

Comment:
perhaps rename to_staging to needs_view_update, so staging knowledge is confined to make_sstables_available().

Reply:
staging is an implementation detail of persisting knowledge of sstables needing view update.